Tests/SentryTests/SentrySessionTests.swift

```diff
@@ -33,4 +33,16 @@ class SentrySessionTestsSwift: XCTestCase {
         let duration = sessionSerialized["duration"] as? Double ?? -1
         XCTAssertEqual(2, duration)
     }
+    
+    func testCopySession() {
+        let session = SentrySession(releaseName: "1.0.0")
+        let copiedSession = session.copy() as! SentrySession
+        
+        XCTAssertEqual(session, copiedSession)
```

Comment:
Maybe we also make sure the reference isn't the same?

```suggestion
        XCTAssertEqual(session, copiedSession)
        XCTAssert(session != copiedSession)
        
```

Reply:
Very good point

Reply:
We already do that in the implementation of equals: https://github.com/getsentry/sentry-cocoa/blob/8ff654e6c5608e4b4d57e0ebf124b78745b3f6cc/Tests/SentryTests/Protocol/SentrySession%2BEquality.m#L8

Reply:
I suggested to add here so it's visible/clear that it isn't the same reference (and remove that from the code would break the tests) but that's a `L` (sorry I didn't add). so np